sources/editor/Stride.Assets.Presentation/TemplateProviders/RenderStageReferenceTemplateProvider.cs

```diff
@@ -17,8 +14,7 @@ public class RenderStageReferenceTemplateProvider : NodeViewModelTemplateProvide
         public override bool MatchNode(NodeViewModel node)
         {
             // Everything that is not in GraphicsCompositorAsset.RenderStages should become a reference
-            return node.Type == typeof(RenderStage)
-                && !(node.Parent?.MemberInfo != null && node.Parent.MemberInfo.DeclaringType == typeof(GraphicsCompositorAsset) && node.Parent.MemberInfo.Name == nameof(GraphicsCompositorAsset.RenderStages));
```

Comment:
I would prefer it if we don't suppress this code without any proper replacement (assuming it was there for a reason).
Either:
- fix it (not sure of exact equivalent of `MemberInfo.DeclaringType`, maybe `Root.Type`?).
- if not sure or difficult to test, I would leave the current broken code in, otherwise we will forget about it if this code is simply suppressed.
- in this specific case, it seems that `RenderStageReferenceTemplateProvider` might not be even used at all? so it might be easier to make sure of that and delete the class if true.

Hopefully, we're in the 3rd case (which would make sense, otherwise the behavior would have been broken).

Same applies for all uses of `MemberInfo`